airflow/www/views.py

```diff
@@ -2108,8 +2237,10 @@ def landing_times(self, session=None):
         )
 
     @expose('/paused', methods=['POST'])
-    @has_dag_access(can_dag_edit=True)
-    @has_access
+    @auth.has_access([
+        (permissions.ACTION_CAN_EDIT, permissions.RESOURCE_DAGS),
```

Comment:
Hmmmmm - I do wonder if long term we'd want more granular permissions on DAGs.

For example, being able to pause/unpause is a lot less "dangerous" than some things we could do on a DAG. (I guess right now there aren't any other edit operations on a DAG)

Reply:
I think that's easy-ish to add later on, or at the point that granularity it needed. If we need it now, can add it though.